src/commands.ts

```diff
@@ -661,7 +661,20 @@ function restartLSPServer(workspaceContext: WorkspaceContext): Promise<void> {
 function reindexProject(workspaceContext: WorkspaceContext): Promise<unknown> {
     return workspaceContext.languageClientManager.useLanguageClient(async (client, token) => {
         try {
-            return await client.sendRequest(reindexProjectRequest, {}, token);
+            await client.sendRequest(reindexProjectRequest, {}, token);
+            const result = await vscode.window.showWarningMessage(
+                "Re-indexing a project should never be necessary and indicates a bug in SourceKit-LSP. Please file an issue describing which symbol was out-of-date and how you got into the state.",
+                "Report Issue",
+                "Close"
+            );
+            if (result === "Report Issue") {
+                vscode.commands.executeCommand(
+                    "vscode.open",
+                    vscode.Uri.parse(
+                        "https://github.com/swiftlang/sourcekit-lsp/issues/new?title=Symbol%20Indexing%20Issue"
```

Comment:
Could we include the template because it prompts for some useful information?

```suggestion
                        "https://github.com/swiftlang/sourcekit-lsp/issues/new?template=BUG_REPORT.yml&title=Symbol%20Indexing%20Issue"
```

Reply:
Nice, I didn't know this was available.